sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/responses/AzureResponsesTest.java

```diff
@@ -144,6 +147,48 @@ public void listInputItems(HttpClient httpClient, AzureResponsesServiceVersion s
         }
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.responses.TestUtils#getTestParametersResponses")
+    public void testResponsesInputContentImageBase64(HttpClient httpClient, AzureResponsesServiceVersion serviceVersion) {
+        ResponsesClient client = getAzureResponseClient(httpClient, serviceVersion);
+
+        try {
+            openImageFileBase64Runner((base64Image) -> {
+                CreateResponsesRequest request = new CreateResponsesRequest(CreateResponsesRequestModel.GPT_4O_MINI,
+                    Arrays.asList(
+                        new ResponsesSystemMessage(Arrays.asList(new ResponsesInputContentText(
+                            "You are a helpful assistant that describes images"))),
+                        new ResponsesUserMessage(Arrays.asList(
+                            new ResponsesInputContentText("Please describe this image"),
+                            new ResponsesInputContentImage().setImageUrl(
+                                "data:image/jpeg;base64," + base64Image)))));
+
+                ResponsesResponse response = client.createResponse(request);
+                assertImageResponseForAzure(response);
+
+            });
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
I am not entirely sure this is a good idea. If the method above throws, then in the `catch` block there isn't any assertion that fails the test. It might be better to change the method signature to `throws IOException` so if it does eventually throw the test fails. Then you wouldn't need the try/catch, I think.